rjc_account_form/reports/invoice_credit_note_layout.xml

```diff
@@ -0,0 +1,204 @@
+<odoo>
+    <data>
+        <template id="rjc_invoice_credit_note_layout_report">
+            <div class="article">
+                <div class="header">
```

Comment:
class `header and footer` should be outside article (header -> article -> footer)